metal/multitask/mt_label_model.py

```diff
@@ -12,39 +13,54 @@ def __init__(self, K=None, task_graph=None, **kwargs):
             K: A t-length list of task cardinalities (overrided by task_graph
```

Comment:
Remove this?  Can be separate PR but I think should have to have `TaskGraph` here

Reply:
Nvm, fine with this